packages/flutter_tools/pubspec.yaml

```diff
@@ -17,7 +17,7 @@ dependencies:
   intl: '>=0.14.0 <0.16.0'
   json_rpc_2: ^2.0.0
   json_schema: 1.0.6
-  linter: 0.1.31
+  linter: 0.1.35
```

Comment:
This will only update the `flutter analyze`.  `flutter analyze --watch` uses analysis server which gets it's linter package w/ the vended dart SDK (see: bin/internal/dart-sdk.version for that).